package.json

```diff
@@ -35,8 +35,6 @@
     "azure-devops-node-api": "^10.2.2",
     "js-yaml": "^3.13.1",
     "minimatch": "3.0.2",
-    "minimist": "^1.2.5",
```

Comment:
NIT: these changes need to be added with a separate PR that has already been created in several quantities

Reply:
So, shall I remove it from this pr?

Reply:
Let's remove

Reply:
removed